gtk/src/rt.rs

```diff
@@ -103,6 +103,7 @@ pub unsafe fn set_initialized() {
 /// so will not cause the program to terminate if GTK could not be initialized.
 /// Instead, an Ok is returned if the windowing system was successfully
 /// initialized otherwise an Err is returned.
+#[doc(alias = "gtk_init_check")]
```

Comment:
not needed, maybe make it  `gtk_init` as it's the right one to use?

Reply:
I'll make a replacement for it.